docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1171,14 +1306,7 @@ function setAudioOutputDevice() {
 }
 
 function toggleSendAudio() {
```

Comment:
there is no visual indication if audio is muted or not

Reply:
Added mute/unmute text on toggle audio button as per microphone's current state.